client/my-sites/hosting/sftp-card/index.js

```diff
@@ -116,41 +124,96 @@ const SftpCard = ( {
 		);
 	};
 
+	const displayQuestionsAndButton = ! ( disabled || username || isLoading );
+
 	return (
 		<Card className="sftp-card">
 			<MaterialIcon icon="cloud" size={ 32 } />
-			<CardHeading>{ translate( 'SFTP Information' ) }</CardHeading>
+			<CardHeading>{ translate( 'SFTP Credentials' ) }</CardHeading>
 			<div className="sftp-card__body">
-				{ disabled || username || isLoading ? (
-					<p>
-						{ translate( "Access and edit your website's files directly using an FTP client." ) }
-					</p>
-				) : (
-					<>
-						<p>
-							{ translate(
-								"Enable SFTP access to generate a username and password so you can access your website's files."
-							) }
-						</p>
-						<Button onClick={ createUser } primary>
-							{ translate( 'Enable SFTP' ) }
-						</Button>
-					</>
-				) }
+				<p>
+					{ translate( "Access and edit your website's files directly by using an SFTP client." ) }
+				</p>
 			</div>
+			{ displayQuestionsAndButton && (
```

Comment:
Starting to get to the point where some child components for the enabled and non-enabled states might simplify this component, but we can look at that when we circle back to allow multiple ftp users.